ci-operator/step-registry/mco-day2/create-custom-mcp/mco-day2-create-custom-mcp-commands.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+
+function validate_params() {
+
+  if  [ "$CUSTOM_MCP_NAME" == "" ]
+  then
+    echo "ERROR: no name has been provided for the custom MCP"
+    exit -1
+  fi
+
+  NUM_WORKERS=$(oc get nodes -l $CUSTOM_MCP_FROM_LABEL -oname | wc -l)
+
+  if [ "$CUSTOM_MCP_NUM_NODES" != "" ] && (( $CUSTOM_MCP_NUM_NODES > $NUM_WORKERS  )) 
```

Comment:
for multi-arch cluster, node number in custom mcp is not related to worker number, I assume all the workers are amd64 based, and node in custom pool is arm64 based. if the rendered mc of worker pool has any amd64 only config, it cannot be applied on nodes in custom pool, so we need label name of node role as input.